python3-poetry.json

```diff
@@ -0,0 +1,259 @@
+{
+  "name": "poetry",
```

Comment:
poetry is needed at build time only? If so you can clean it up using `"cleanup": [ "*" ]`

Reply:
done